app/services/create_github_repo_service/stats_sender.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+# Wrapper class over GitHubClassroom.statsd for creating repositories
+# This enables us to add extra stats based on the progress of the repository being created
+#
+class CreateGitHubRepoService
+  class StatsSender
+    class InvalidStatError < StandardError; end
+    attr_reader :exercise
+    def initialize(exercise)
```

Comment:
Can you add a newline after `class InvalidStatError < StandardError; end` and `attr_reader :exercise`